src/js/pages/nodes/NodeDetailPage.js

```diff
@@ -0,0 +1,373 @@
+                {resourceValue}
+              </h4>
+              <span className={`side-panel-resource-label
+                  text-color-${colorIndex}`}>
+                {resourceLabel.toUpperCase()}
+              </span>
+            </div>
+          </div>
+        </div>
+
+        <Chart calcHeight={function (w) { return w / WIDTH_HEIGHT_RATIO; }}
+          delayRender={true}>
+          <BarChart
+            axisConfiguration={axisConfiguration}
+            data={resourceData}
+            inverseStyle={true}
+            margin={{top: 0, left: 43, right: 10, bottom: 40}}
+            maxY={maxY}
+            refreshRate={Config.getRefreshRate()}
+            ticksY={3}
+            xGridLines={0}
+            y="percentage" />
+        </Chart>
+      </div>
+    );
+  }
+
+  getSubHeader(node) {
+    let activeTasksCount = node.sumTaskTypesByState('active');
+    let activeTasksSubHeader = StringUtil.pluralize('Task', activeTasksCount);
+    let healthStatus = node.getHealth();
+
+    return (
+      <ul className="list-inline flush-bottom">
+        <li>
+          <span className={healthStatus.classNames}>
+            {healthStatus.title}
+          </span>
+        </li>
+        <li>
+          {`${activeTasksCount} Active ${activeTasksSubHeader}`}
+        </li>
+      </ul>
+    );
+  }
+
+  renderHealthTabView() {
+    let node = this.internalStorage_get().node;
+    let units = NodeHealthStore.getUnits(node.hostname);
+
+    return (
+      <div className="container container-fluid flush">
+        <HealthTab
+          node={node}
+          units={units}
+          parentRouter={this.context.router} />
+      </div>
+    );
+  }
+
+  renderTasksTabView() {
+    let nodeID = this.props.params.nodeID;
+    let tasks = MesosStateStore.getTasksFromNodeID(this.props.params.nodeID);
+
+    let contents = this.getLoadingScreen();
+
+    let timeSinceMount = (Date.now() - this.mountedAt) / 1000;
+    if (timeSinceMount >= SidePanelContents.animationLengthSeconds) {
+      contents = (
+        <TaskView
+          inverseStyle={true}
+          tasks={tasks}
+          parentRouter={this.context.router}
+          nodeID={nodeID} />
+      );
+    }
+
+    return (
+      <div className="container container-fluid flush">
+        {contents}
+      </div>
+    );
+  }
+
+  renderDetailsTabView() {
+    let nodeID = this.props.params.nodeID;
+    let last = MesosSummaryStore.get('states').lastSuccessful();
+    let node = last.getNodesList().filter({ids: [nodeID]}).last();
+
+    if (node == null) {
+      return null;
+    }
+
+    let headerValueMapping = {
+      ID: node.id,
+      Active: StringUtil.capitalize(node.active.toString().toLowerCase()),
+      Registered: DateUtil.msToDateStr(
+        node.registered_time.toFixed(3) * 1000
+      ),
+      'Master Version': MesosStateStore.get('lastMesosState').version
+    };
+
+    return (
+      <div className="container container-fluid flush">
+        <DescriptionList
```

Comment:
Just checked with Canvas and this just how dark the inverse styles for text is: http://mesosphere.github.io/canvas/#typography-inverse-styling